src/App.css

```diff
@@ -36,3 +36,35 @@
     transform: rotate(360deg);
   }
 }
+
+.locationButton {
+  margin-top: 20px;
+}
+
+div.RepBlock + h2 {
+  font-size: 40px;
+  justify-content: left;
+}
+
+.RepCard.primary {
```

Comment:
You don't need to duplicate all the css for primary and secondary, you could have one .rep-card class with all the css here, and a .primary and .secondary

Reply:
These should also probably be moved to the RepCard component